pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -444,21 +444,41 @@ func (m *kubeGenericRuntimeManager) computePodContainerChanges(pod *v1.Pod, podS
 	}
 	changes.InitFailed = initFailed
 
+	podRef, err := ref.GetReference(api.Scheme, pod)
+	if err != nil {
```

Comment:
(This has since changed to containerRef to reference the specific container in the event instead of the pod, but...)

Yes, but Eventf tolerates this.  I'm not sure what the best error handling would be here, we certainly don't want to bail out but I definitely wouldn't want future developers using podRef in this scope assuming it's non-nil.

I saw this in a few other places in the codebase (not bailing out if a ref used for Events isn't found), so I carried the behavior forward.